server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/rfc/IdTest.scala

```diff
@@ -0,0 +1,68 @@
+/** **************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ * *
+ * http://www.apache.org/licenses/LICENSE-2.0                 *
+ * *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ * ***************************************************************/
+package org.apache.james.jmap.rfc
+
+import org.apache.james.jmap.rfc.model.Id.Id
+import org.scalatest.{Matchers, WordSpec}
+import play.api.libs.json.{JsError, JsString, JsSuccess, Json}
+import eu.timepit.refined.auto._
+import be.venneborg.refined.play.RefinedJsonFormats._
+import com.fasterxml.jackson.databind.exc.MismatchedInputException
+
+class IdTest extends WordSpec with Matchers {
```

Comment:
@remk : can you help checking this test class with Id being Refined? I'm not sure it's enough and/or correct